frontend/src/main/scala/bloop/exec/Forker.scala

```diff
@@ -212,7 +212,10 @@ object Forker {
       @tailrec
       def traverseLines(start: Int): Unit = {
         if (start < msg.length) {
-          val lineEnd = msg.indexOf(System.lineSeparator(), start)
+          val lineEnd = msg.indexOf("\r\n", start) match {
```

Comment:
```suggestion
          val (lineEnd, lineEndLength) = msg.indexOf("\r\n", start) match {
            case -1 => (msg.indexOf("\n", start), 1)
            case idx => (idx, 2)
          }
```
?

Reply:
I like this change, although I assume you also intended to remove the recalculation of lineEndLength on line 217.